### DISCUSSION TITLE
Running proj vs. using the SS3 forecast.ss file

### DISCUSSION THREAD

**Discussion by JaneSullivan-NOAA** [afsc-assessments/AK-projection-simulation#14] [General]
@zandjyo a place for future discussions on this topic.

General questions (feel free to add more):
1. What stock assessments use the forecast.ss file for proj? Is it all the SS3 assessments or are there some SS3 assessments that use proj?
2. Has any testing been done to identify differences between proj and SS3? Are there known differences between the methods? 
3. Can the forecast.ss file be set up for partial assessment years?
4. Can the forecast.ss file be set up to alter future catches (e.g., if catch << ABC)?

**Comment by zandjyo** [comment]
1) All the cod models, I don't think any others.
2) Yes, not well documented, but was examined in the EBS and GOA Cod models. Not well documented outside of the September PT reports. Should probably revisit this at some point.
3) Yes, specify the catch in the forecast.ss. My projections script was not really set up for partials, but could easily be fixed.
4 ) Yes and in the Projections for SS script I have a method where this can be specified either by F or by tons reading in a .csv
